exawind/Pazy_Wing/README.md

```diff
@@ -0,0 +1,177 @@
+
+The CFD computational domain consists of an inner Nalu-Wind domain immediately surrounding the wing, and an outer AMR-Wind domain to allow space for the wake to propagate.  The sides, ceiling, and floor of the AMR-Wind domain are intended to approximate the effect of the wind tunnel walls used in the experiment.  However, to avoid the computational expense of resolving the boundary layers near the tunnel walls, *slip wall* boundaries are used there.  The Nalu-Wind domain extends a few chord-lengths away from the wing, and terminates with an *overset* boundary on the exterior, and a *slip wall* on the portion that touches the bottom wind tunnel boundary.  The inner boundary of the Nalu-Wind domain represents the wing surface, and uses a *no-slip wall* boundary condition.
+
+In the experiment, the blade tip passes near the wind-tunnel walls at peak deflection.  Since the Nalu-Wind domain extends several chord lengths away from the wing, the AMR-Wind domain was chosen to be somewhat larger than the wind tunnel in order to avoid collisions of the overset boundary with the slip wall at peak deflection, and additional padding was provided to ensure the results are not contaminated by the effect of the artificial slip wall boundaries.  Future studies are planned to investigate the impact of the choice of AMR-Wind domain size on the simulation results.
+
+**Grid Generation**
+
+<img src="figures/mesh_overview_100.png" border="0" alt=""/>
+
+The Nalu-Wind grid was generated using Pointwise, and a cut-away view is shown in the left figure above.  The wing tip surface grid used an O-grid surrounding an H-grid, to which elliptic smoothing was applied.  The outer boundary of this wing tip grid was extruded to yield the complete surface mesh, shown in red above.  Finally, the volume mesh was created through hyperbolic extrusion of the surface mesh.  The grid for AMR-Wind was generated using its internal grid generator, and has one refinement region in the interior, centered around the wing.  This grid is shown in the right figure above.  Metrics for these grids are given in the table below.
+
+| Nalu-Wind Grid Property         | Value                     |   | AMR-Wind Grid Property   | Value                  |
+| :---                            | ---                       | - | :---                     | ---                    |
+| Number of cells                 | 8,736,600                 |   | Domain size              | 3 m x 3 m x 1.5 m      |
+| First cell height               | 1.0E-06 m                 |   | Level 0 dimensions       | 128 x 128 x 64         |
+| First cell y+                   | 0.2                       |   | Number of cells          | 5,767,168              |
+| Cells around airfoil            | 500                       |   | Level 0 spacing          | ~0.024 m               |
+| Number of spanwise layers       | 100                       |   | Level 1 spacing          | ~0.012 m               |
+
```

Comment:
Should we specify that the same Nalu-Wind grid was used for all wind speeds and AOA sweeps in the study?

Reply:
I added some information to make this clear.  I also added that the y+ corresponds to the highest wind speed, which was something I had forgotten to mention.